WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -135,7 +135,7 @@ local function GetBuffTriggerOptions(data, optionTriggerChoices)
       hidden = function() return not (trigger.type == "aura2" and trigger.unit ~= "multi" and not IsSingleMissing(trigger)) end
     },
     debuffClass = {
-      type = "select",
+      type = "multiselect",
       width = WeakAuras.normalWidth,
       name = L["Debuff Type"],
```

Comment:
Can we go ahead and add a new feature indicator?

Reply:
on first or second title ?
![img](https://i.imgur.com/4dIUEnI.png)

Reply:
Ah, the first one.